### ISSUE TITLE
[BUG] SfImage invalid html issue

### ISSUE THREAD

**Issue by DavidGolodetsky** [vuestorefront/storefront-ui#2104]
**Describe the bug**
Nuxt HTML-validator throws an error on the SfSearchBar component:

`Attribute "srcset" has invalid value ""`

**How to reproduce**
1. Create a component, without `srcset` prop being passed
2. Put the output of the component to the [W3C validator](https://validator.w3.org/#validate_by_input+with_options)

**Expected behavior**
The SfImage component restructured in a way that HTML would be valid

**Actual behavior**
The SfImage component has a `srcset` prop, when the value not passed it results in an empty `scrset` HTML attribute

**Screenshots**

![image](https://user-images.githubusercontent.com/32278933/141277969-b46d6440-cc84-45a8-b82e-776fcd6979cc.png)

![image](https://user-images.githubusercontent.com/32278933/141279181-938573bb-b5db-41bb-9168-6b6518708063.png)

**Additional context**
Related to #2097

**Possible solution**
The `srcset` prop seems to be unnecessary since we already have v-bind="$attrs" directive there. If the srcset HTML attribute was set from the parent it should appear in the place of the current `srcset` prop

**Comment by DavidGolodetsky** [comment]
Could be assigned to me

**Comment by justyna-13** [comment]
@DavidGolodetsky how is it going with this issue? Do you need some help? :)

**Comment by DavidGolodetsky** [comment]
sorry for the delay, I'll open a PR today:)

**Comment by DavidGolodetsky** [comment]
Strange, but I cannot reproduce it anymore. Probably it was due to our misuse of the component. Sorry for bothering, I'm closing the issue:)
